src/code/z_parameter.c

```diff
@@ -2992,52 +2995,37 @@ void func_8008A994(InterfaceContext* interfaceCtx) {
     func_800AB2C4(&interfaceCtx->view);
 }
 
-s16 sMagicArrowEffectsR[] = { 255, 100, 255 };
-s16 sMagicArrowEffectsG[] = { 0, 100, 255 };
-s16 sMagicArrowEffectsB[] = { 0, 255, 100 };
-
-s16 sTimerDigitLeftPos[] = { 16, 25, 34, 42, 51 };
-s16 sDigitWidth[] = { 9, 9, 8, 9, 9 };
-
-// unused, most likely colors
-s16 D_80125B1C[][3] = {
-    { 0, 150, 0 }, { 100, 255, 0 }, { 255, 255, 255 }, { 0, 0, 0 }, { 255, 255, 255 },
-};
-
-s16 sRupeeDigitsFirst[] = { 1, 0, 0 };
-s16 sRupeeDigitsCount[] = { 2, 3, 3 };
-
-s16 sSpoilingItemEntrances[] = { 0x01AD, 0x0153, 0x0153 };
-
-u16 D_80125B54 = 0xC220;       // unused
-u16 D_80125B58 = 0xC20C;       // unused
-s16 D_80125B5C[] = { 91, 91 }; // unused
-
-// Due to an unknown reason, bss ordering changes within the 5 static variables in the function below.
-// In order to restore the correct order, we need a specific number of bss variables in the file before that point.
-// For this, we introduce 2 dummy variables which end up in padding at the end of the file's bss, so they don't actually
-// take space.
-s8 sBssDummy1;
-s8 sBssDummy2;
-
 #ifdef NON_MATCHING
-// mostly regalloc, minor ordering and stack usage differences
```

Comment:
Any chance the comment could be updated to describe what the remaining issues are that makes it nonmatching?